src/System.Net.Ping/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Net.NetworkInformation.Tests
     public class LoggingTest
     {
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot, "NetEventSource is only part of .NET Core and requires internal Reflection")]
```

Comment:
/cc @stephentoub 
I think the comment is referring only to the test itself, not ETW logging.
I just tested that I'm able to capture ETW traces from a UWP application with the current code and https://github.com/dotnet/corefx/blob/master/Documentation/debugging/windows-instructions.md#traces